src/settings/config.rs

```diff
@@ -22,13 +25,6 @@ fn neovide_config_dir() -> PathBuf {
     xdg_dirs.get_config_home().unwrap()
 }
 
-#[cfg(windows)]
-fn neovide_config_dir() -> PathBuf {
-    let mut path = dirs::config_dir().unwrap();
```

Comment:
Why is only the the windows specific part moved out, while the other platforms remain? I don't think moving that elsewhere makes the code any clearer.